src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.sharedfx.targets

```
@@ -251,7 +251,7 @@
     generate shared frameworks in the Arcade SDK.
   -->
   <Target Name="BuildPkgProjectReferences">
-    <MSBuild Projects="$(RepoRoot)signing\SignBinaries.proj" Targets="Build" />
+    <MSBuild Projects="$(SigningToolsDirectory)\SignBinaries.proj" Targets="Build" />
```

Redundant `\` (and applying `Dir` change):

```suggestion
    <MSBuild Projects="$(SigningToolsDir)\SignBinaries.proj" Targets="Build" />
```